torch/_dynamo/polyfills/builtins.py

```diff
@@ -58,3 +59,8 @@ def enumerate(iterable: Iterable[_T], start: int = 0) -> Iterable[tuple[int, _T]
 @substitute_in_graph(builtins.sum, can_constant_fold_through=True)  # type: ignore[arg-type]
 def sum(iterable: Iterable[_T], /, start: _T = 0) -> _T:  # type: ignore[assignment]
     return functools.reduce(operator.add, iterable, start)
+
+
+@substitute_in_graph(keyword.iskeyword, can_constant_fold_through=True)  # type: ignore[arg-type]
+def iskeyword(s: str) -> bool:
+    return s in keyword.kwlist
```

Comment:
I'd prefer to move this to a separate `polyfills/keyword.py` file. Because `keyword` is a separate module in stdlib.